src/borg/helpers/manifest.py

```diff
@@ -53,7 +56,7 @@ def __getitem__(self, name):
         _name = safe_encode(name)
         values = self._archives.get(_name)
         if values is None:
-            raise KeyError
+            raise NoArchiveError
```

Comment:
this is a good idea to support wide-scope catching, but not needed here, see above.